From 92313eada1617b25df3346e45f9c103888919e83 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <uwu@icenowy.me>
Date: Tue, 17 Dec 2024 01:35:14 +0800
Subject: [PATCH 078/314] FROMLIST: mips: disable CRASH_DUMP by default

On MIPS, the space of a crash dump kernel needs to be manually specified
by both the debugee and the crash dump kernel PHYSICAL_START config
option, and the default PHYSICAL_START config will make the kernel
load quite higher, which isn't acceptable when using as a daily use
kernel (and will even confuse some naive Loongson-3 bootloaders).

So I don't think a MIPS kernel should be built as a crash dump kernel by
default, therefore this patch disables selecting CRASH_DUMP by default.

Signed-off-by: Icenowy Zheng <uwu@icenowy.me>

Link: https://lore.kernel.org/all/20241216173514.495638-1-uwu@icenowy.me/
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/Kconfig | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index 5078ebf071ec..efcaca5f4820 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -2875,9 +2875,6 @@ config ARCH_SUPPORTS_KEXEC
 config ARCH_SUPPORTS_CRASH_DUMP
 	def_bool y
 
-config ARCH_DEFAULT_CRASH_DUMP
-	def_bool y
-
 config PHYSICAL_START
 	hex "Physical address where the kernel is loaded"
 	default "0xffffffff84000000"
-- 
2.48.1

